scout/server/blueprints/variant/templates/variant/clinvar.html

```diff
@@ -560,14 +581,21 @@ <h5>
                                   {% if sample.display_name == individual.display_name %}
                                     <strong>({{ sample.genotype_call }})</strong><br>
                                     <select name="zygosity@{{pinned._id}}" id="zygosity">
-                                      {% if sample.genotype_call == '1/1' %}
+                                      {% if sample.genotype_call == '1/1' and pinned.chromosome in ['X','Y'] %}
```

Comment:
Isn't there a more elegant way to add that selected tag? Seems a little repetitive..

Reply:
There sure is, jinja style!